doc/ref_model/chapters/chapter07.rst

```diff
@@ -708,33 +709,91 @@ Access to the repository must be granted by a dedicated authorization. The code
+components. The SBOM is an inventory of software components and the relationships between them. NTIA describes how to 
+establish an SBOM and provides SBOM standard data formats. In case of vulnerability detected for a component, the 
+SBOM inventory is an effective means to identify the impacted component and provide remediation.
+
+A transparent software supply chain offers benefits for vulnerabilities remediation, but also for licensing management 
+and it provides assurance of the source and integrity of components. To achieve and take benefit of this transparency, 
+a shared model must be supported by industry. This is the goal of the work performed by the USA department of commerce 
+and the National Telecommunications and Information administration (NTIA) and published, in July 2021, in the report 
+`“The Minimum Elements for a Software Bill of Materials (SBOM)” <https://www.ntia.doc.gov/files/ntia/publications/sbom_\
+minimum_elements_report.pdf>`_. The document gives guidance and describes sets the minimum elements for the SBOM, as 
+a starting point. 
+
+A piece of software can be modeled as a hierarchical tree with components and subcomponents, each 
+component should have its SBOM including,  as a baseline, the information described in the following table.
+
+
++------------------------------+---------------------------------------------------+
+| **Data Field**               | Description                                       |
++==============================+===================================================+
+| **Supplier Name**            | The name of an entity that creates, defines, and  |
+|                              | identifies components.                            |
++------------------------------+---------------------------------------------------+
+| **Component Name**           | Designation assigned to a unit of software        |
+|                              | defined by the original supplier.                 |
++------------------------------+---------------------------------------------------+
+| **Version of the Component** | Identifier used by the supplier to specify a      |
+|                              | change in software from a previously identified   |
+|                              | version.                                          |
++------------------------------+---------------------------------------------------+
+| **Other Unique Identifiers** | Other identifiers that are used to identify a     |
+|                              | component, or serve as a look-up key for relevant |
+|                              | databases.                                        |
++------------------------------+---------------------------------------------------+
+| **Dependency Relationship**  | Characterizing the relationship that an upstream  |
+|                              | component X is included in software Y.            |
```

Comment:
Is it software or a component? I realise that software is also a comonent

Reply:
The same as the previous answer; It is the exact definition of NTIA document.
A software is composed of multiple components, these components are software